### PR TITLE
Document SG Rest Post endpoint

### PR DESCRIPTION
**Pull request by best-ankitagarwal**

### CODE REVIEW

**Comment by bringdev** [comment]
[trello.com/c/bM9bi3bL](https://trello.com/c/bM9bi3bL)

File: api/shipping-guide_2/api.raml [hunk 1]
```diff
@@ -483,6 +484,29 @@ documentation:
         displayName: Unauthorized
         description: |
           A failed response when the request is unauthenticated
+  post:
+    description: Get shipment prices and leadtime
+    headers:
+      X-MyBring-API-Uid:
```
Comment (tomgag) [inline api/shipping-guide_2/api.raml:490]: What about that `X-Bring-Client-URL`. It seems to not be required, even if the docs indicate so.

File: api/shipping-guide_2/examples/rest_estimated_delivery_time_request.json [hunk 2]
```diff
@@ -0,0 +1,55 @@
+{
+  "consignments": [
+    {
+      "id": 1,
+      "products": [
+        {
+          "id": "SERVICEPAKKE",
```
Comment (tomgag) [inline api/shipping-guide_2/examples/rest_estimated_delivery_time_request.json:7]: Maybe use a revised service?

File: api/shipping-guide_2/examples/rest_estimated_delivery_time_request.json [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+          "customerNumber": "PARCELS_NORWAY-00000000001"
```
Comment (tomgag) [inline api/shipping-guide_2/examples/rest_estimated_delivery_time_request.json:8]: Do we usually anonymize customer numbers?

Comment (best-ankitagarwal) [reply api/shipping-guide_2/examples/rest_estimated_delivery_time_request.json:8]: this is a dummy customer.

File: api/shipping-guide_2/api.raml [hunk 4]
```diff
@@ -327,9 +327,10 @@ documentation:
       For cargo services(5100, 5300, 9100, CARGO), Weight is mandatory and one of the Volume / Load Meter / Dimensions / Number of Pallets is mandatory.
       
       Note: The prices for our revised services depends on volumetric weight: 5000, 5100, 5300, 5600, 5800, 4850 plus return services 9000, 9100, 9300 or 9600. For these services, dimensions (L/W/H) is needed to calculate the expected price correctly. Dimensions are also used to decide if handling fee or specialgoods fee will apply to the shipment. You might leave dimensions empty in your request, but be aware that returned prices might be incorrect.
+      Get endpoint will be deprecated soon. Preffered way of using it is Post request.
```
Comment (tomgag) [inline api/shipping-guide_2/api.raml:330]: Suggestions:

The GET endpoint

The preferred

**Review by tomgag — APPROVED** [review]
Couple of suggestions, otherwise looks 👍

File: api/shipping-guide_2/examples/rest_estimated_delivery_time_request.json (hunk 3 above)
Comment (tomgag) [reply api/shipping-guide_2/examples/rest_estimated_delivery_time_request.json:8]: oka

File: api/shipping-guide_2/api.raml (hunk 1 above)
Comment (best-ankitagarwal) [reply api/shipping-guide_2/api.raml:490]: so @tomgag  should we add it as part of documentation and treat as optional? As of now it is not covered

Comment (tomgag) [reply api/shipping-guide_2/api.raml:490]: No idea, I see we usually don't require it, but devsite says it is needed.

**Pull request merged**